Caramelo18/Visual-Kur
# Save file

## What was added
1. Saving the file edited to the original file

## How to Test
1. Open the app
2. Pick a file 
3. Change file in a text editor 
4. Click on the "Save" button
5. Verify the original file was chenged
